pkg/client/unversioned/extensions.go

```diff
@@ -130,24 +130,24 @@ func NewExtensionsOrDie(c *Config) *ExtensionsClient {
 
 func setExtensionsDefaults(config *Config) error {
 	// if experimental group is not registered, return an error
-	g, err := latest.Group("extensions")
-	if err != nil {
-		return err
+	if !extlatest.IsEnabled() {
```

Comment:
> not sure switching to using package vars for client setup will be better we get to populating from discovery... for the client defaults, I sort of like having one spot where we retrieve group info. for most of the other places (server setup, etc) the package dependency makes sense

And if that's what it did before, that's what it would still do.  Hardcoding the group you're checking means, "I want that one", not "I want to lookup something like this opaque name".  When they make it generic, this can be generic.